### ISSUE TITLE
Quickstart page "Enable API" button returns API key and not credentials.json

### ISSUE THREAD

**Issue by drakeboy77** [gsuitedevs/python-samples#129]
## Expected Behavior
Was expecting a download button to initiate a download of the credentials.json file after Google had activated my Calendar API

## Actual Behavior
![API_KEY_ISSUE](https://user-images.githubusercontent.com/24443195/65394286-60584d80-dd51-11e9-8930-c2b0213fe254.PNG)

## Steps to Reproduce the Problem

1. I have gone back to page https://developers.google.com/calendar/quickstart/python several times and every time it only returns the API key and no credentials.json
1.
1.

## Specifications

- Python version (`python --version`) 2.7
- OS (Mac/Linux/Windows) Ubuntu 18.04

**Comment by epoz** [comment]
Yup, also baffled by this behaviour in https://developers.google.com/sheets/api/quickstart/python 
It looks like the documentation does not match the resulting behaviour in the Cloud Console.

How do you generate/download a valid `credentials.json` file if you only have an API key?

**Comment by carkod** [comment]
## Solution

After some research I was able to run the quickstart.py without changing a single line, here is how to get the credentials.json

1. Go to https://console.developers.google.com/
2. Create a project
![create_project](https://user-images.githubusercontent.com/14939793/65598336-79ecd580-df9b-11e9-975d-17cc07b75b8d.png)
3. Add API and services
![add_apis](https://user-images.githubusercontent.com/14939793/65598418-a4d72980-df9b-11e9-955b-8e938d116117.png)
4. Enable Gmail API
![enable_gmail_api](https://user-images.githubusercontent.com/14939793/65598457-c0dacb00-df9b-11e9-8288-c9550bbcc126.png)
5. Create a client ID
![create_oauth_client_id](https://user-images.githubusercontent.com/14939793/65598529-eff13c80-df9b-11e9-8bee-4c40291a2cc5.png)
6. Download credentials
![download_oauth_credentials](https://user-images.githubusercontent.com/14939793/65598552-00a1b280-df9c-11e9-930c-455001f2731f.png)
7. Now, copy and paste the contents of this downloaded .json into your credentials.json

**Comment by tenleftfingers** [comment]
The option to download is no longer available. A file could be crafted manually if the correct format / key-value pair syntax were known.
